empress/__init__.py

```diff
@@ -0,0 +1,102 @@
+"""
+Wraps empress functionalities
+"""
+import matplotlib.pyplot as plt
+from typing import List, Iterable
+from abc import ABC, abstractmethod
+from empress.xscape.CostVector import CostVector
+
+class Drawable(ABC):
+    """
+    Implements a default draw method
+    """
+
+    @abstractmethod
+    def draw_on(self, axes: plt.Axes):
+        """
+        Draw self on matplotlib Axes
+        """
+        pass
+
+    def draw(self) -> plt.Figure:
+        """
+        Draw self as matplotlib Figure.
+        """
+        # create a new figure
+        # draw on the figure
+        # return the figure
+        pass
+
+    def draw_to_file(self, fname):
+        """
+        Draw self and save it as pdf at path fname.
+        """
+        figure = self.draw()
+        figure.savefig(fname)
+
+class ReconInputWrapper:
+    def __init__(self, parasite_tree: dict, host_tree: dict, tip_mapping: dict):
+        self._parasite_tree = parasite_tree
+        self._host_tree = host_tree
+        self._tip_mapping = tip_mapping
+
+class ReconciliationWrapper(Drawable):
+    # TODO: Replace dict with Reconciliation type
+    # https://github.com/ssantichaivekin/eMPRess/issues/30
+    def __init__(self, reconciliation: dict):
+        self._reconciliation = reconciliation
+
+    def draw_on(self, axes: plt.Axes):
+        pass
+
+class ReconGraphWrapper(Drawable):
+    # TODO: Replace dict with ReconGraph type
+    # https://github.com/ssantichaivekin/eMPRess/issues/30
+    def __init__(self, recongraph: dict):
+        self._recongraph = recongraph
+
+    def find_median(self) -> ReconciliationWrapper:
+        """
+        Find and return one median of self
+        """
+        pass
+
+    def draw_on(self, axes: plt.Axes):
+        pass
+
+    def cluster(self, n) -> List['ReconGraphWrapper']:
+        """
+        Cluster self into n reconciliation graphs
+        """
+        pass
+
+class CostPolygonWrapper(Drawable):
```

Comment:
Not sure what this one does, but I'd call this CostRegionWrapper rather than CostPolygonWrapper

Reply:
ok. It's a wrapper that draws the costscape window :)

Reply:
renamed! :) (see new file changes!)